cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/TablesCreator.java

```diff
@@ -297,6 +298,23 @@ public void createMessagesTestEventsTable() throws IOException
 		exec.executeQuery(create.asCql(), true);
 		logger.info("Table '{}' has been created", tableName);
 	}
+
+	public void createHealingIntervalsTable() throws IOException
+	{
+		String tableName = settings.getHealingIntervalsTableName();
+		if (isTableExists(tableName))
+			return;
+
+		CreateTable create = SchemaBuilder.createTable(settings.getKeyspace(), tableName).ifNotExists()
+				.withPartitionKey(INSTANCE_ID, DataTypes.UUID)
+				.withPartitionKey(HEALING_INTERVAL_ID, DataTypes.TEXT)

```

Comment:
With this table you can write interval data, but to get interval you'll need to know its ID. I doubt if crawlers actually know these IDs, they only know timestamp intervals to work in.